### DISCUSSION TITLE
How to use the `doc` shortcode?

### DISCUSSION THREAD

**Discussion by Xaldew** [getnikola/nikola#3786] [Q&A]
Hi!

I'm trying to use the `doc` shortcode to create a link to another post in an org-mode or markdown, but I don't want the link spell out the name of the post. That is, I really want to do the same thing listed in the first example [here](https://getnikola.com/handbook.html#doc). However, the obvious translation of the rst code to do this:

```
{{% doc %}}post <creating-a-theme>{{% /doc %}}
```

Doesn't seem to work, and just yields an "invalid-link" error. Am I using the shortcode incorrectly, or is this behavior not implemented?

https://getnikola.com/handbook.html#shortcodes

**Comment by Kwpolska** [comment]
This should work:

    {{% doc data='post <creating-a-theme>' %}}

**Reply by Xaldew** [reply]
Hmmm. I did some tests on a markdown file:

```
Works: {{% doc %}}clang-format{{% /doc %}}
Works: {{% doc %}}random-text content <clang-format>{{% /doc %}}
Doesn't work: {{% doc data='post <clang-format>' %}}...
```

Last one yields: `[2024-10-29 23:13:30] ERROR: Nikola: "" slug doesn't exist.`
